Debian builds fail with commit:    plugins: use the short form of peekProc

Commit 921ee19e3abd24482921d85b38c0c4b3d0cbcffe throws a compiler error on Debian Stretch.

`g++ -m64 -Wl,-z,relro -Wl,-z,now -Wl,-O1 -shared -o liblink.so release/link-posix.o  -L/usr/lib/x86_64-linux-gnu/c++11 -lrt  
In file included from ../mumble_plugin_linux_32bit.h:14:0,
                 from l4d2_linux.cpp:6:
../mumble_plugin_linux.h: In instantiation of ‘bool peekProc(procptr32_t, T&) [with T = unsigned char; procptr32_t = unsigned int]’:
l4d2_linux.cpp:26:41:   required from here
../mumble_plugin_linux.h:188:15: error: invalid conversion from ‘unsigned char’ to ‘void*’ [-fpermissive]
  out.iov_base = dest;
  ~~~~~~~~~~~~~^~~~~~
mv -f liblink.so ../../release/plugins/`

--- a/plugins/mumble_plugin_linux.h
+++ b/plugins/mumble_plugin_linux.h
@@ -185,7 +185,7 @@ bool peekProc(PTR_TYPE base, T &dest) {
 	in.iov_len = sizeof(T); // Length
 
 	struct iovec out;
-	out.iov_base = dest;
+	out.iov_base = &dest;
 	out.iov_len = sizeof(T);
 
 	ssize_t nread = process_vm_readv(pPid, &out, 1, &in, 1, 0);

